code/game/objects/items/food/burgers.dm

```diff
@@ -86,25 +113,33 @@
 	name = "roburger"
 	desc = "The lettuce is the only organic component. Beep."
 	icon_state = "roburger"
-	food_reagents = list(/datum/reagent/consumable/nutriment = 8, /datum/reagent/cyborg_mutation_nanomachines = 6, /datum/reagent/consumable/nutriment/vitamin = 6)
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 8,
+		/datum/reagent/cyborg_mutation_nanomachines = 6,
+		/datum/reagent/consumable/nutriment/vitamin = 6,
+	)
 	tastes = list("bun" = 4, "lettuce" = 2, "sludge" = 1)
 	foodtypes = GRAIN | TOXIC
 	venue_value = FOOD_PRICE_EXOTIC
 
-/obj/item/food/burger/roburgerbig
-	name = "roburger"
+/obj/item/food/burger/roburger/big
```

Comment:
Is there a ``/obj/item/food/burger/roburger``?

Reply:
Yup! It should be the object just above the big roburger.